contrib/pyln-client/pyln/client/plugin.py

```diff
@@ -283,7 +289,7 @@ def get_option(self, name):
         else:
             return self.options[name]['default']
 
-    def async_method(self, method_name, category=None, desc=None, long_desc=None):
+    def async_method(self, method_name, category=None, desc=None, long_desc=None, deprecated=False):
```

Comment:
You missed the corresponding `add_method` below for this one

Reply:
Yes, missed method() here too.  Fixed.